src/main/java/mate/controller/IndexController.java

```diff
@@ -1,9 +1,36 @@
 package mate.controller;
 
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 import mate.dao.MyCoolResource;
 
+@WebServlet(urlPatterns = "/")
 public class IndexController extends HttpServlet {
     private MyCoolResource myResource;
-    //TODO: implement
+
+    @Override
+    public void init() {
+        myResource = MyCoolResource.openResource();
+    }
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+                                    throws ServletException, IOException {
+        String currentDateTime =
+                LocalDateTime.now().format(DateTimeFormatter.ofPattern("d MMM yyyy HH:mm:ss"));
+        req.setAttribute("date", currentDateTime);
```

Comment:
why do you setAttribute and don't use in the jsp. Either use it or don't pass at all. According to the task you need to write date and time only to resource. You can do it in one line and withour pattern

Reply:
done